framework/include/wifi_manager/wifi_manager.h

```diff
@@ -116,7 +124,7 @@ typedef struct wifi_manager_scan_info_s wifi_manager_scan_info_s;
  */
 typedef struct {
 	void (*sta_connected)(wifi_manager_result_e);	// in station mode, connected to ap
-	void (*sta_disconnected)(void);		// in station mode, disconnected from ap
+	void (*sta_disconnected)(wifi_manager_disconnect_to);		// in station mode, disconnected from ap
```

Comment:
If the public callback API changes, how about change it all together in tizenRT?

Reply:
Yes, we have decided to change the public callback API of Wi-Fi Manager to provide the information of re(auto)connect. Based on the modification, I changed all corresponding disconnect callback function in various apps and framework such as st_things.

Reply:
Oh yes,
I have not seen the second commit.